hre/src/main/java/hre/io/MessageProcessEnvironment.java

```diff
@@ -87,4 +86,15 @@ public MessageProcess startProcess() {
 
     return new MessageProcess(workingDirectory, argv.toArray(new String[0]), env);
   }
+
+  public String getInvocation() {
+    StringBuilder sb = new StringBuilder();
+    sb.append(process);
+    sb.append(" ");
```

Comment:
this seems suspect, surely strings with spaces should be quoted?

Reply:
Since it's only intended for quick debugging of the messageprocessenvironment, and currently unused, I'll delete the method instead.